website/docs/reference/resource-configs/athena-configs.md

```diff
@@ -0,0 +1,621 @@
+
+- `insert_overwrite` (default): The insert-overwrite strategy deletes the overlapping partitions from the destination table and then inserts the new records from the source. This strategy depends on the `partitioned_by` keyword! dbt will fall back to the `append` strategy if no partitions are defined.
+- `append`: Insert new records without updating, deleting or overwriting any existing data. There might be duplicate data (great for log or historical data).
+- `merge`: Conditionally updates, deletes, or inserts rows into an Iceberg table. Used in combination with `unique_key`.It is only available when using Iceberg.
+
+
+### On schema change
+
+The `on_schema_change` option reflects changes of the schema in incremental models. The values you can set this to are:
+
+- `ignore` (default)
+- `fail`
+- `append_new_columns`
+- `sync_all_columns`
+
+To learn more, refer to [What if the columns of my incremental model change](/docs/build/incremental-models#what-if-the-columns-of-my-incremental-model-change).
+
+### Iceberg
+
+The adapter supports table materialization for Iceberg.
+
+For example:
+
+```sql
+{{ config(
+    materialized='table',
+    table_type='iceberg',
+    format='parquet',
+    partitioned_by=['bucket(user_id, 5)'],
+    table_properties={
+     'optimize_rewrite_delete_file_threshold': '2'
+     }
+) }}
+
+select 'A'          as user_id,
+       'pi'         as name,
+       'active'     as status,
+       17.89        as cost,
+       1            as quantity,
+       100000000    as quantity_big,
+       current_date as my_date
+```
+
+Iceberg supports bucketing as hidden partitions. Use the `partitioned_by` config to add specific bucketing
+conditions.
+
+Iceberg supports these table formats for data : `PARQUET`, `AVRO` and `ORC`.
+
+To use Iceberg incrementally, use one of the following supported strategies:
+
+- `append`: New records are appended to the table (this can lead to duplicates).
+- `merge`: Perform an update and insert (and optional delete), where new and existing records are added. It is only available with Athena engine version 3.
+  - `unique_key`(required): Columns defining a unique source and target table record.
+  - `incremental_predicates` (optional): SQL conditions that enable custom join clauses in the merge statement. This can
+    help improve performance via predicate pushdown on the target table.
+  - `delete_condition` (optional): SQL condition used to identify records that should be deleted.
+  - `update_condition` (optional): SQL condition used to identify records that should be updated.
+  - `insert_condition` (optional): SQL condition used to identify records that should be inserted.
+    - `incremental_predicates`, `delete_condition`, `update_condition` and `insert_condition` can include any column of the incremental table (`src`) or the final table (`target`). Column names must be prefixed by either `src` or `target` to prevent a `Column is ambiguous` error.
```

Comment:
```suggestion
   - `incremental_predicates`, `delete_condition`, `update_condition` and `insert_condition` can include any column of the incremental table (`src`) or the final table (`target`). Column names must be prefixed by either `src` or `target` to prevent a `Column is ambiguous` error.
```